src/routing_table/mod.rs

```diff
@@ -869,6 +890,26 @@ impl<T: Binary + Clone + Copy + Debug + Default + Hash + Xorable> RoutingTable<T
         Ok(())
     }
 
+    #[cfg(any(test, feature = "use-mock-crust"))]
+    pub fn our_name(&self) -> &T {
+        &self.our_name
+    }
+
+    /// Returns the list of contacts as a result of a merge to which we aren't currently connected,
+    /// but should be.
+    #[cfg(any(test, feature = "use-mock-crust"))]
+    pub fn needed(&self) -> &Groups<T> {
+        &self.needed
+    }
+
+    #[cfg(test)]
+    pub fn verify_invariant(&self) {
+        if let Err(_) = self.check_invariant() {
+            let message = format!("Invariant not satisfied for RT: {:?}", self);
+            panic!(message);
```

Comment:
Let's print the error in this panic message, too.
(I know it's only GitHub highlighting this code as changed, but still. :grin: )